demos/python_demos/whiteboard_inpainting_demo/utils/network_wrappers.py

```diff
@@ -0,0 +1,123 @@
+                         mode='constant', constant_values=0).reshape(1, self.c, self.h, self.w)
+        im_info = im_info.reshape(1, *im_info.shape)
+        output = self.net.net.infer(dict(im_data=im_data, im_info=im_info))
+
+        classes = output['classes']
+        valid_detections_mask = classes > 0
+        classes = classes[valid_detections_mask]
+        boxes = output['boxes'][valid_detections_mask]
+        scores = output['scores'][valid_detections_mask]
+        masks = output['raw_masks'][valid_detections_mask]
+        return boxes, classes, scores, np.full(len(classes), 0, dtype=np.int32), masks
+
+    def get_detections(self, frames, return_cropped_masks=False, only_class_person=True):
+        outputs = []
+        for frame in frames:
+            data_batch = self.preprocess(frame)
+            im_data = data_batch['im_data']
+            im_info = data_batch['im_info']
+            meta = data_batch['meta']
+
+            boxes, classes, scores, _, masks = self.forward(im_data, im_info)
+            scores, classes, boxes, masks = postprocess(scores, classes, boxes, masks,
+                                                        im_h=meta['original_size'][0],
+                                                        im_w=meta['original_size'][1],
+                                                        im_scale_y=meta['processed_size'][0] / meta['original_size'][0],
+                                                        im_scale_x=meta['processed_size'][1] / meta['original_size'][1],
+                                                        full_image_masks=True, encode_masks=False,
+                                                        confidence_threshold=self.confidence)
+            frame_output = []
+            for i in range(len(scores)):
+                if only_class_person and classes[i] != 1:
+                    continue
+                bbox = [int(value) for value in boxes[i]]
+                if return_cropped_masks:
+                    left, top, right, bottom = bbox
+                    mask = masks[i][top:bottom, left:right]
+                else:
+                    mask = masks[i]
+                frame_output.append([bbox, scores[i], mask])
+            outputs.append(frame_output)
+        return outputs
+
+
+class SemanticSegmentation:
+    def __init__(self, ie, model_path, conf=.6, device='CPU', ext_path=''):
+        self.confidence = conf
+        self.net = load_ie_model(ie, model_path, device, None, ext_path, num_reqs=1)
+
+    def get_detections(self, frames, only_class_person=True):
+        outputs = []
+        for frame in frames:
+            out_h, out_w = frame.shape[:-1]
+            res = self.net.forward(frame)
+            output = []
+            for batch, data in enumerate(res):
+                data = data.transpose((1, 2, 0)).astype('uint8')
+                data = cv2.resize(data, (out_w, out_h))
+                data = np.where(data > 9, 1, 0)
```

Comment:
What is `9`?

Reply:
Logic was reimplemented with list of named classes which will be hided using text files with labels according to a model.